graph-subscriptions-api/Cargo.toml

```
@@ -26,7 +25,8 @@ reqwest = { version = "0.11.15", default-features = false, features = [
     "trust-dns",
 ] }
 serde = "1.0.158"
-serde_json = "1.0.94"
+serde_json = { version = "1.0.94", features = ["raw_value"] }
```

I don't think you need the `raw_value` feature
```suggestion
serde_json = "1.0.94"
```